### PR TITLE
Feat: Added add token feature

### PR DESCRIPTION
**Pull request by RyRy79261**
### Description

This PR adds the hooks to allow users to add the Mento tokens to their wallets by clicking the token balances in the connected drop down menu

## Related issues

- [#450](https://github.com/mento-protocol/mento-general/issues/450)

### CODE REVIEW

File: src/lib/hooks/useAddTokens.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+              chainId !== Celo.id || chainId !== Alfajores.id
+                ? Celo.id
+                : chainId
+            ].MentoToken, // The address that the token is at.
+          symbol: "MENTO", // A ticker symbol or shorthand, up to 5 chars.
+          decimals: 18, // The number of decimals in the token
+        },
+      },
+    });
+    console.log(`Mento token added: ${wasAdded}`);
```
Comment (chapati23) [inline src/lib/hooks/useAddTokens.ts:28]: do you want to keep this console log in here

File: src/lib/hooks/useAddTokens.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+              chainId !== Celo.id || chainId !== Alfajores.id
+                ? Celo.id
+                : chainId
+            ].Locking, // The address that the token is at.
+          symbol: "veMENTO", // A ticker symbol or shorthand, up to 5 chars.
+          decimals: 18, // The number of decimals in the token
+        },
+      },
+    });
+    console.log(`veMento token added: ${wasAdded}`);
```
Comment (chapati23) [inline src/lib/hooks/useAddTokens.ts:49]: console.log out?

**Review by chapati23 — COMMENTED** [review]
works for me, just a question on lingering console.logs in the code

File: src/lib/hooks/useAddTokens.ts (hunk 1 above)
Comment (RyRy79261) [reply src/lib/hooks/useAddTokens.ts:28]:
```suggestion
```

File: src/lib/hooks/useAddTokens.ts (hunk 2 above)
Comment (RyRy79261) [reply src/lib/hooks/useAddTokens.ts:49]:
```suggestion
```

**Review by chapati23 — APPROVED** [review]
lgtm

**Review by philbow61 — APPROVED** [review]
nice one 💯

**Pull request merged**